frontend/packages/integration-tests-cypress/reporter-config.json

```
@@ -1,7 +1,14 @@
 {
-  "reporterEnabled": "spec, mocha-junit-reporter",
+  "reporterEnabled": "mocha-junit-reporter, mochawesome",
   "mochaJunitReporterReporterOptions": {
     "mochaFile": "../../gui_test_screenshots/junit_cypress-[hash].xml",
     "toConsole": false
+  },
+  "mochawesomeReporterOptions": {
+    "reportDir": "../../gui_test_screenshots/",
+    "reportFilename": "junit_cypress",
```

If this is for the JSON/HTML files, we don't need this prefix. `junit_` here is misleading since it's not a junit file. Maybe

```suggestion
    "reportFilename": "cypress-report",
```

---

Since we specify `"html": false`, this is just for the JSON files.  I can change the name.

---

We don't need the JSON files for testgrid, only the JUnit XML report. Deleting is fine.